feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -374,10 +382,19 @@ func TestTrafficWithGracefulRestartSpeaker(t *testing.T) {
 		t.Log("Start DUT interface Config")
 		configureDUT(t, dut)
 	})
+
+	// Configure Network Instance on the DUT
```

Comment:
Can you move this block to the **configureDUT()** function

Reply:
Done, thank you